### ISSUE TITLE
Is there some kind of hack again? how can the net 53% pass?

### ISSUE THREAD

**Issue by kityanhem** [gcp/leela-zero#1467]
![image](https://user-images.githubusercontent.com/22489225/40340274-4ab9244c-5da8-11e8-81a4-7252d7a5dacb.png)

**Comment by l1t1** [comment]
in some point it beyond 0.55, then lost many games, so drop the winrate

**Comment by kityanhem** [comment]
Yeah drop the winrate. I thought it still pass with the text fail. Oh, I see. Now ever weight pass has pass even lower than 55%.

**Comment by l1t1** [comment]
you copy the vs page to a excel sheet, then input function in a new column
=COUNTIF($B$2:B2,"7c6588ce")/COUNTA($B$2:B2)*100, then you will find the change of winrate
